content/blog/TinaCMS-Version-21.mdx

```diff
@@ -9,30 +9,24 @@ We are excited to announce the release of `tinacms@2.1.0` and `@tinacms/cli@1.5.
 
 ### What's New:
 
-1. **Removed Deprecated Hook**
+1. **API - Removed Deprecated Hook**
    * A deprecated hook, which was part of legacy internal systems, has now been fully removed. This change does not affect the commonly used `useTina` hook.
```

Comment:
feels odd that these are bullet points, when there is only a single item in each section - recommend removing them

Reply:
Fixed - 
![image](https://github.com/user-attachments/assets/779dd8e2-8d51-413c-9638-4a38cb25b7ec)